File: pyt/removeDups.py
"""

Create a function that takes a list of items,

removes all duplicate items and returns a new list in the same sequential order as the old list (minus duplicates).

"""




def remove_dups(lst):
    seen = []
    for item in lst:
        if item not in seen:
            seen.append(item)

    return seen

File: c#/removeDups.cs
    public static object[] RemoveDups(object[] str) 
    /* Create a function that takes an array of items, removes all duplicate items and returns a new array in the same sequential order as the old array (minus duplicates).*/
    {

        object[] result = str.Distinct().ToArray();

        return result;


      
    }